### PR TITLE
Calculate homework

### PR DESCRIPTION
**Pull request by kondorski**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: calculate
🏅 Granted 6 XP for @kondorski

**Comment by ziobron** [comment]
⏱ 3 XP extra for @kondorski for on-time delivery

File: homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -1,7 +1,31 @@
...
-std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
-}
+#pragma once
+#include <string>
+
+std::string calculate(const std::string& command, int first, int second) {
+    
+    // TODO: Implement your solution here and return proper value
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:6]: remove

**Review by nauka-programowania-MA — APPROVED** [review]
Try to commit only your changes :) I noticed that you modify a lot of files.

**Pull request closed without merging**